src/components/ContentGroups/CampaignThanks.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+	<div class="small-12 large-8 align-self-middle columns">
+		<div class="small-12 columns thanks">
+			<template v-if="loans.length > 0">
+				<div class="thanks__header hide-for-print">
+					<h1 class="thanks__header-h1">
+						Thank you!
+					</h1>
+					<p class="thanks__header-subhead">
+						Thanks for supporting {{ borrowerSupport }}.<br>
+						<span class="hide-for-print">
+							We've emailed your order confirmation to {{ lender.email }}
+						</span>
+					</p>
+				</div>
+			</template>
+		</div>
+
+		<template v-if="loans.length > 0">
+			<social-share
+				class="thanks__social-share"
+				:lender="lender"
+				:loans="loans"
+			/>
+		</template>
+
+		<div class="small-12 columns thanks">
+			<hr v-if="loans.length > 0">
+			<checkout-receipt
+				v-if="showReciept"
+				class="thanks__receipt"
+				:lender="lender"
+				:receipt="receipt"
+			/>
+		</div>
+	</div>
+</template>
+
+<script>
+import confetti from 'canvas-confetti';
+
+import CheckoutReceipt from '@/components/Checkout/CheckoutReceipt';
+import SocialShare from '@/components/Checkout/SocialShare';
+import thanksPageQuery from '@/graphql/query/thanksPage.graphql';
+import { joinArray } from '@/util/joinArray';
+
+export default {
+	components: {
+		CheckoutReceipt,
+		SocialShare,
+	},
+	inject: ['apollo'],
+	metaInfo() {
+		return {
+			title: 'Thank you!'
+		};
+	},
+	props: {
+		transactionId: {
+			type: Number,
+			default: null,
+		},
+	},
+	data() {
+		return {
+			lender: {},
+			loans: [],
+			receipt: {},
+			showReciept: false,
+		};
+	},
+	computed: {
+		borrowerSupport() {
+			const loanNames = this.loans.map(loan => loan.name);
+			if (loanNames.length > 3) {
+				return `these ${loanNames.length} borrowers`;
+			}
+			return joinArray(loanNames, 'and');
+		}
+	},
+	mounted() {
+		if (this.transactionId) {
+			this.fetchReceipt();
+		}
+	},
+	methods: {
+		fetchReceipt() {
+			this.apollo.query({
+				query: thanksPageQuery,
+				variables: {
+					checkoutId: this.transactionId
+				}
+			}).then(({ data }) => {
+				this.lender = {
+					...data.my.userAccount,
+					teams: data.my.teams.values.map(value => value.team)
```

Comment:
can we guarantee this will always exist here? Maybe change to optional chaining?

Reply:
I could probably add some more checks... This came right out of the thanks page. I just moved it out of the prefetch structure.